Week2/akif.js

```diff
@@ -0,0 +1,108 @@
+// Part 1 Hello World
+console.log('Hello World'); // English
+console.log('Merhaba Dunya'); // Turkish
+console.log('Hallo Wereld'); // Dutch
+console.log('مرحبا بالعالم'); // Arabic
+
+// Part 2 Error  -----------------------------------------
+console.log("I'm awesome");
+
+// Part 3 Console.log(x)   --------------------------------
+let x;
+console.log('the value of my variable x will be: 16');
```

Comment:
Hi Akif - the value of `x` here would be `undefined` because we haven't assigned it a value on line 11. It's only a declaration. It is only at line 14 when we assign `x` with a value. 🙂